lib/livebook/session/data.ex

```diff
@@ -1727,6 +1741,17 @@ defmodule Livebook.Session.Data do
     set!(data_actions, notebook: %{data.notebook | hub_secret_names: hub_secret_names})
   end
 
+  defp update_notebook_deployment_group({data, _} = data_actions, hub) do
+    hub_type = Livebook.Hubs.Provider.type(hub)
+
+    deployment_group =
+      if hub_type == "team", do: Livebook.Teams.get_deployment_groups(hub) |> List.first()
```

Comment:
We should not hardcode to Livebook.Teams here. We probably need to add a new function to the Provider. Perhaps a `default_deployment_group_id`? And for the personal hub is simply returns nil?

Reply:
Also, do we need to set a default group id? Or should we just default to nil regardless of hub?

Reply:
I set a default if there are deployment groups available, otherwise nil. But it's up to you! We can also always default to nil. wdyt?

Reply:
Let’s default to nil. It is just less work for now and we figure it out later.

Reply:
Ok!